sdk/storage/storage-blob/.vscode/launch.json

```diff
@@ -0,0 +1,51 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Samples",
```

Comment:
We have samples for "js" and "ts". Will them both be covered?

Reply:
I've just added config for js samples as well.